lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -29,16 +29,32 @@ $(document).ready(function() {
   // Enable the Onboarding experiment.
   if (Drupal.settings.dsOnboarding.enabled && localStorage.getItem(localStorageKey) !== 'true') {
     const jsxElementHook = 'jsx-onboarding';
-    const slides = [InstructionSlide]; // ReportbackItemKudosSlide, ContextSlide
+    const slides = [];
```

Comment:
Yeah it's kinda confusing. You can safely use `const` here since you're not changing the binding of what variable is assigned to `slides` (just adding/removing objects to that same array).